doc/src/manual/embedding.md

```diff
@@ -284,6 +284,50 @@ args[1] = some_other_value;
 JL_GC_POP();
 ```
 
+Each scope must have only one call to `JL_GC_PUSH*`. Hence, if it is necessary to protect more
+variables inside the same function, one can use inner blocks:
```

Comment:
More conventional method is to just use a larger frame, i.e. `JL_GC_PUSH2` here.

Reply:
I will make this clear and change to something like "If it is necessary to call it more than once...".